### DISCUSSION TITLE
Theme development with Zig - Orchard Core Pair Programming by Lombiq

### DISCUSSION THREAD

**Discussion by Piedone** [OrchardCMS/OrchardCore#15955] [Show and tell]
We'll have the first session of Orchard Core Pair Programming by Lombiq! In these, we do an hour of pair programming with an Orchard Core community member about a project of theirs. We learn together a lot, share best practices, and write some good code. All this is live, and you can join us with your questions! Check it out here: https://www.youtube.com/watch?v=RdE6Io4XiZ8. See you at 5 PM UTC on the 6th!

This time, [Ermir Pellumbi aka Zig](https://github.com/cptzig) will join us with a theme project. Your host and the "navigator" of the pair programming session will be me.

**Comment by Piedone** [comment]
We're live! https://www.youtube.com/watch?v=RdE6Io4XiZ8

**Reply by hishamco** [reply]
I think the session is short :) we might create a continuation to finalize the theme. It would be nice if Zig could start from an HTML theme and convert it to be utilized from OC

**Reply by Piedone** [reply]
We thought that nobody can pay attention, us included, for more than an hour :D. But we can continue with this next time indeed, or take a different approach as you mention.

**Comment by Piedone** [comment]
Thank you everyone for taking part, this was nice :). The recording will be available shortly. Would you like us to continue with this theme project, or do something else? And let me know if you'd like to be a guest with your own project!

**Reply by hishamco** [reply]
> And let me know if you'd like to be a guest with your own project!

Let me check the client if he never mind I can share what I did in a side project, but for now I might prefer to let Zig finalize a theme in another session using Liquid or he can use media theme, then me or anyone else can do theming with Razor

**Reply by Piedone** [reply]
Let me know, them. You're most definitely welcome too!

**Reply by hishamco** [reply]
Zoltan the company never mind showing their website, can we talk about the details here or shall we move this conversation via email

**Reply by Piedone** [reply]
Awesome! Would you like to discuss this on Discord? Under https://orchardcore.net/discord.

**Reply by hishamco** [reply]
I didn't try it, but I will have a look

**Reply by MarGraz** [reply]
**I'm looking for a person who commented in the YouTube chat during the live stream** with the profile name "TinyScale". 

They responded to me about the template and **invited me to open a discussion**. I've already started a discussion, but I can't find them as a user here. **TinyScale**, if you are reading this, you can find the already opened discussion [here](https://github.com/OrchardCMS/OrchardCore/discussions/15464) 🤣 This is a message in a bottle.

**Comment by Piedone** [comment]
And now the recording is out: https://www.youtube.com/watch?v=RdE6Io4XiZ8.
